utils/diff_model_converter.py

```diff
@@ -345,6 +348,11 @@ def replace_call_to_super(class_finder: ClassFinder, updated_node: cst.ClassDef,
             func = func.with_changes(body=updated_methods[name].body, params=new_params)
         end_meth.append(func)
 
+    # Port new methods that are defined only in diff-file and append at the end
+    for name, func in updated_methods.items():
+        if name not in original_methods and func is not None and isinstance(func, cst.FunctionDef):
+            end_meth.append(func)
+
```

Comment:
I don't really think this should matter as you use `updated_methods` in the new_module.visit with super transformers

Reply:
This func returns `original_node.with_changes()` at the end which is further modified by other methods. Without this line two new methods are not ported from diff to the model, because original node doesn't have it

Reply:
Gottcha. I checked and you are right! 
I should have added a test where there is a new function!